gradle-plugin/kubernetes/src/main/java/org/eclipse/jkube/gradle/plugin/task/AbstractJKubeTask.java

```diff
@@ -131,6 +132,7 @@ protected final KitLogger createLogger(String prefix) {
             .reactorProjects(Collections.singletonList(kubernetesExtension.javaProject))
             .sourceDirectory(kubernetesExtension.getBuildSourceDirectoryOrDefault())
             .outputDirectory(kubernetesExtension.getBuildOutputDirectoryOrDefault())
+            .namespace(applicableNamespace(null, kubernetesExtension.getNamespaceOrNull(), kubernetesExtension.resources, clusterAccess))
```

Comment:
This will require cluster access for Resource and Helm tasks, I'm not sure if we want to follow that path.